docs/specs/error.yml

```diff
@@ -155,3 +155,43 @@ outputs:
     {"message_severity":"info","code":"exceed-max-file-errors","message":"Info count exceed '2'. Build will continue but newer logs in 'a.md' will be ignored.","file":"a.md","line":0,"end_line":0,"column":0,"end_column":0}
 
 ---
+# override by buildRules - system error
+noDryRun: true
+inputs:
+  docfx.yml: |
+    markdownValidationRules: rule.json
+    buildValidationRules: build.json
+  rule.json: |
+    {
+      "images": {
+        "rules": [
+          {
+            "type": "ImageAltTextEqualsFilename",
+            "message": "AltText {0} is the same as the image file name.",
+            "code": "altText-equal-filename",
+            "severity": "SUGGESTION"
+          }
+        ]
+      }
+    }
+  build.json: |
+    {
+      "system": {
+        "rules": [
+          {
+            "type": "FileNotFound",
+            "message": "Invalid file link: '{0}'. Pls Check.",
```

Comment:
Have to use this kind of string template like `{0}, {1}` in ruleset. In generated Errors.cs we can still use `{name}` style.

Reply:
it is ok after talked with yufei